src/services/lnd/index.ts

```diff
@@ -245,12 +245,7 @@ export const LndService = (): ILightningService | LightningServiceError => {
           : undefined,
         max_fee: maxFee,
         payment: decodedInvoice.paymentSecret || undefined,
-        total_mtokens: decodedInvoice.paymentSecret
-          ? decodedInvoice.milliSatsAmount > 0
-            ? decodedInvoice.milliSatsAmount.toString()
-            : (amount * 1000).toString()
-          : undefined,
-        tokens: amount,
```

Comment:
this was the main issue, `ProbeForRouteArgs` uses `MergeExclusive` for `mtokens` and `tokens` so now we can't set both